File: TuristickaAgencija.Model/Putovanja.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public partial class Putovanja
    {
        public int Id { get; set; }
        public string NazivPutovanja { get; set; }
        public string OpisPutovanja { get; set; }
        public float CijenaPutovanja { get; set; }
        public DateTime DatumPolaska { get; set; }
        public DateTime DatumDolaska { get; set; }
        public int BrojMjesta { get; set; }
        public byte[] Slika { get; set; }
        public int GradId { get; set; }
        public int PrevozId { get; set; }
        public int SmjestajId { get; set; }

        public ICollection<VodiciPutovanja> VodiciPutovanja { get; set; }
    }
}

File: turisticka_agencija_mobile/lib/models/Putovanja.dart
import 'VodiciPutovanja.dart';

import 'package:json_annotation/json_annotation.dart';

part 'Putovanja.g.dart';

@JsonSerializable()
class Putovanja {
    int brojMjesta;
    double cijenaPutovanja;
    String datumDolaska;
    String datumPolaska;
    int gradId;
    int id;
    String nazivPutovanja;
    String opisPutovanja;
    int prevozId;
    String slika;
    int smjestajId;
    List<VodiciPutovanja> vodiciPutovanja;

    Putovanja({required this.brojMjesta,required this.cijenaPutovanja,required this.datumDolaska,
        required this.datumPolaska,required this.gradId,required this.id, required this.nazivPutovanja,
        required this.opisPutovanja,required this.prevozId,required this.slika,required this.smjestajId,
        required this.vodiciPutovanja});

    factory Putovanja.fromJson(Map<String, dynamic> json) => _$PutovanjaFromJson(json);

    Map<String, dynamic> toJson() => _$PutovanjaToJson(this);
}